app/javascript/packages/countdown/countdown-element.spec.ts

```diff
@@ -1,4 +1,5 @@
 import sinon from 'sinon';
+import { waitFor } from '@testing-library/react';
```

Comment:
Should there be any changes to this file if we haven't modified the implementation of `<lg-countdown-element>` ?

Reply:
Good catch. I was messing around in different files and missed this when I was cleaning up the file.